src/poprox_recommender/recommenders.py

```diff
@@ -32,7 +36,7 @@ def select_articles(
     pipeline state whose ``default`` is the final list of recommendations.
     """
     available_pipelines = recommendation_pipelines(device=default_device())
-    pipeline = available_pipelines["nrms"]
+    pipeline = available_pipelines["cali"]
```

Comment:
This will change the default when no recommender is specified in the request from plain NRMS to NRMS+Locality calibration. That's probably okay if you plan to run this experiment from a separate endpoint, but maybe not something we want to do in the main/default recommender endpoint

Reply:
This raises a good point. Are we expected to fork the repo and run the fork? 

Is POPROX planning to have a release cycle so we can uptake changes to the main POPROX?

Reply:
Current plan is to have people fork the repo and deploy from the fork. No current plans for a defined release cycle since it hasn't become an issue yet.

Reply:
Certainly! So based on the conversation looks like it's OK to keep the pipeline as `cali` instead of `nrms` if we will run only run this with allocated participants?

I'm actually also thinking about the possibility of running some experiment flag to determine which pipeline do we want to choose here:

```
if interest_profile.experiment_id == LOCALITY_EXPT_ID:
    pipeline = available_pipelines["cali"]
else:
    pipeline = available_pipelines["nrms"]
```

But I realized that might not be the way we want to allocate users based on experiment manifest?

Reply:
By the time the endpoint defined in this repo receives a request, it will already be for the appropriate recommender for each user according to the group assignments. The section that defines the experiment recommenders in the manifest will look something like this:
```toml
[recommenders.calibration]
endpoint = "https://n07svs3qeh.execute-api.us-east-1.amazonaws.com/?pipeline=calibration"
```
The platform side sends requests for users assigned to groups that use that recommender to that URL, and the query parameter at the end of the URL gets used by [this code](https://github.com/CCRI-POPROX/poprox-recommender/blob/main/src/poprox_recommender/recommenders.py#L34-L39) to select the appropriate pipeline.

(It's set up that way so that the recommender code doesn't need to know anything about experiments, groups, phases, etc in order to keep it simple for experimenters.)